CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Add `ChatChannelController.addMembers(_ members: [MemberInfo])` [#3487](https://github.com/GetStream/stream-chat-swift/pull/3487)
 - Add `CurrentUserController.updateMemberData()` [#3487](https://github.com/GetStream/stream-chat-swift/pull/3487)
 - Exposes `ChatChannelMember.memberExtraData` property [#3487](https://github.com/GetStream/stream-chat-swift/pull/3487)
+- Add bounce moderation v2 support [#3492](https://github.com/GetStream/stream-chat-swift/pull/3492)
```

Comment:
Feels like "bounce moderation v2" phrasing could be improved.  Should it be just "Add moderation v2 support" or "Add moderation v2 support with bounced messages"?

I looked for general v2 moderation docs but did not find them at the moment. Otherwise it would be excellent to have a link here what the v2 is about.

Reply:
Discussed internally 👍